integrations/outlook/src/misc/custom-schemas.ts

```diff
@@ -0,0 +1,52 @@
+import * as z from "zod";
+
+export const notificationContentSchema = z
```

Comment:
This schema feels a bit complex to use. May I suggest something a bit simpler? 
```js
const emailAddress = z.string().min(1, 'Email cannot be blank')
const emailOrNameEmail = z.union([
  emailAddress, 
  z.object({
    email: emailAddress,
    name: z.string().optional()
  })
]) // so you can just give an array of strings, or also an object with the name

const schema = z.object({
  from: emailOrNameEmail,
  to: z.array(emailOrNameEmail),
  cc: z.array(emailOrNameEmail),
  message: z.string().min(1)
})
```

It's easy to use a string for simplicity and also have an object for more complex setups